test/cypress/dashboards/integration/sync/highlight-options.cy.js

```diff
@@ -0,0 +1,94 @@
+describe("Highlight sync options", () => {
+    before(() => {
+        cy.visit('/dashboards/sync/sync-highlight-options');
+    });
+
+    it('All highlight sync options are disabled', () => {
+        cy.boardRendered();
+
+        cy.get('#showTooltip').click();
+        cy.get('#showMarker').click();
+        cy.get('#showCrosshair').click();
+
+        cy.get('.highcharts-datagrid-cell').eq(0).trigger('mouseover');
+
+        cy.chart().then(chart => {
+            assert.ok(
+                chart.tooltip.isHidden,
+                'When hovering over DataGrid, chart should not have tooltip.'
+            );
+
+            assert.notOk(
+                chart.series[0].points[0].state === 'hover',
+                'When hovering over DataGrid, chart should not have marker.'
+            );
+
+            assert.notOk(
+                chart.xAxis[0].cross && chart.yAxis[0].cross.opacity === 1,
```

Comment:
You can use optional chaining here. `chart.yAxis[0].cross?.opacity === 1` will be enough.